code/Tasks/SiteTreeMaintenanceTask.php

```diff
@@ -14,22 +15,23 @@ class SiteTreeMaintenanceTask extends Controller
 
     public function makelinksunique()
     {
-        $badURLs = "'" . implode("', '", DB::query("SELECT URLSegment, count(*) FROM SiteTree GROUP BY URLSegment HAVING count(*) > 1")->column()) . "'";
-        $pages = DataObject::get("SilverStripe\\CMS\\Model\\SiteTree", "\"SiteTree\".\"URLSegment\" IN ($badURLs)");
+        $table = DataObject::singleton(SiteTree::class)->baseTable();
+        $badURLs = "'" . implode("', '", DB::query("SELECT \"URLSegment\", count(*) FROM \"$table\" GROUP BY \"URLSegment\" HAVING count(*) > 1")->column()) . "'";
+        $pages = DataObject::get(SiteTree::class, "\"$table\".\"URLSegment\" IN ($badURLs)");
 
         foreach ($pages as $page) {
             echo "<li>$page->Title: ";
             $urlSegment = $page->URLSegment;
             $page->write();
             if ($urlSegment != $page->URLSegment) {
                 echo _t(
-                    'SilverStripe\\CMS\\Model\\SiteTree.LINKSCHANGEDTO',
+                    SiteTree::class . '.LINKSCHANGEDTO',
```

Comment:
Unfortunately this doesn’t work on the text collector yet, it’ll need to stay the way it was

Reply:
Change has been undone.